packages/amplify-category-notifications/src/multi-env-manager.ts

```diff
@@ -300,33 +289,32 @@ function writeBackendConfig(context, pinpointMeta, backendConfigFilePath) {
     const jsonString = JSON.stringify(backendConfig, null, 4);
```

Comment:
```suggestion
    const jsonString = JSONUtilities.stringify(backendConfig, null, 4);
```

Reply:
JSONUtilities doesn't seem to accept the same parameters, will leave as is in this move to TS